requirements/conda.txt

```diff
@@ -2,4 +2,3 @@ numpy
 diffpy.utils
 pandas
 scipy
-gooey
```

Comment:
Removed gooey from `conda.txt` since we won't be able to install it and it will only fail the CI during conda install.

Reply:
I think we need to add wxpython to conda.txt

Reply:
yes, done.